test/model.jl

```diff
@@ -738,3 +738,23 @@ facts("[model] Test getValue on OneIndexedArrays") do
     @fact typeof(x) --> Vector{Variable}
     @fact typeof(getValue(x)) --> Vector{Float64}
 end
+
+facts("[model] Relaxation keyword argument to solve") do
+    m = Model()
+    @defVar(m, 1 <= x <= 2, SemiCont)
+    @defVar(m, 1.5 <= y <= 2, Int)
+    @defVar(m, z, Bin)
+    @defVar(m, 0.5 <= w <= 0.5, Int)
+    @defVar(m, 1 <= v <= 2)
+    @defVar(m, -2 <= t <= -1, SemiInt)
+
+    @setObjective(m, Min, x + y + z + w + v - t)
+    solve(m, relaxation=true)
```

Comment:
For simplicity, what about setting ``internalModelLoaded=false`` after a solve with ``relaxation=true``?